### PR TITLE
Added: new rule `value-list-max-empty-lines`.

### PR DESCRIPTION
**Pull request by evilebottnawi**
Close https://github.com/stylelint/stylelint/issues/1713
/cc @davidtheclark @jeddy3

### CODE REVIEW

File: src/rules/value-list-max-empty-lines/index.js [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+    const validOptions = validateOptions(result, ruleName, {
+      actual: max,
+      possible: isNumber,
+    })
+    if (!validOptions) { return }
+
+    root.walkDecls(decl => {
+      if (
+        !isStandardSyntaxDeclaration(decl)
+        || !isStandardSyntaxProperty(decl.prop)
```
Comment (davidtheclark) [inline src/rules/value-list-max-empty-lines/index.js]: Why are these checks needed? Is there anything about non-standard syntax declarations or props that would trip up this rule? If not, let's check them like everything else.

**Comment by davidtheclark** [comment]
LGTM except for the ignoring of non-standard syntax that I'm not sure is necessary here.

**Comment by davidtheclark** [comment]
👍

File: src/rules/value-list-max-empty-lines/README.md [hunk 2]
```diff
@@ -0,0 +1,108 @@
+# value-list-max-empty-lines
+
+Limit the number of adjacent empty lines within value lists.
+
+```css
```
Comment (jeddy3) [inline src/rules/value-list-max-empty-lines/README.md]:
```css
a {
  box-shadow:
    inset 0 2px 0 #dcffa6,
                    /* ← */
    0 2px 5px #000; /* ↑ */
}                   /* ↑ */
/**                    ↑
 *       This empty line */
```

The default `indentation` setting for values is one level more than the declarations. This updates the example to match that.

**Comment by jeddy3** [comment]
If you can just tweak the leading example, then this LGTM.

**Comment by jeddy3** [comment]
LGTM @evilebottnawi Thanks! :)

**Pull request merged**